sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentOptions.cs

```diff
@@ -23,9 +23,9 @@ internal AnalyzeSentimentOptions(TextAnalyticsRequestOptions options)
         }
 
         /// <summary>
-        /// If set to true, response will contain Opinion Mining sentiment analysis results.
-        /// Only available for Text Analytics Service version v3.1-preview.1 and above.
+        /// Specifies the types of analysis to apply, like for example,
+        /// Opinion mining.
         /// </summary>
-        public bool IncludeOpinionMining { get; set; } = false;
-    }
+        public AnalyzeSentimentType IncludeAnalysis { get; set; }
```

Comment:
Some comments on naming here:

- It's a little confusing to say "include analysis" when the method the options is passed into is called "AnalyzeSentiment" -- will it only analyze the sentiment if this has a non-`None` value?  How should users think about the difference between standard analysis and extra analysis like opinion mining or emotion based analysis?

- `Include` usually implies a boolean - either true or false (I include it or I don't), so it's kind of confusing to have a variable called include that's a flags enum and can represent multiple things.  It feels like it'd be more conventional to call this property something like `SentimentAnalysisType` (and maybe name the enum the same way), and it can be set to None, e.g. 
```
await client.AnalyzeSentimentAsync(document, options: new AnalyzeSentimentOptions() { SentimentAnalysisType = SentimentAnalysisType.None });
```

- Or maybe naming it something like `AdditionalSentimentAnalysis` to indicate it's further analysis beyond what you get from a standard call to AnalyzeSentiment?

Reply:
ahh I see your point. Definitely want to make sure the user understand this is additional analysis and that they can ask for more than one.
Good point about the `Include`.
I will think more about the names

Reply:
I updated the PR with `AdditionalSentimentAnalyses` . I am making it plural as it is a Flags enum